dashboard/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "redpanda-dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
we should have static dependencies and fixed versions - `11.1.1` vs `^11.1.1`

also is there a build system that freezes the dependencies to a known checksum? 

we should check in the checksum files too. 

last we should list the licenses on `licenes/third_party.md`

Reply:
If there's a `package-lock.json`, fixed versions are "optional", since npm will check that file to download the deps at those specific versions.

Reply:
@senior7515  I think we can change it to  `~` which allow patch versions (~1.2.3 will allow from 1.2.3 to <1.3.0) this changes are not breaking changes (optimization and bug fixes)